SimTKcommon/include/SimTKcommon/internal/Pathname.h

```diff
@@ -167,6 +190,17 @@ class SimTK_SimTKCOMMON_EXPORT Pathname {
         return absPath;
     }
 
+    static std::string getAbsolutePathnameUsingSpecifiedWorkingDirectory(const std::string& swd,
```

Comment:
Let's call this `findAbsolute ...` rather than `getAbsolute ...`. Simbody's convention is that `get` means "things you can get for free with no calculation", that is, some kind of accessor. Some other verb could work to, like `construct`.